WEAPON/CreateWeaponObject.md

```diff
@@ -5,26 +5,21 @@ ns: WEAPON
 
 ```c
 // 0x9541D3CF0D398F36 0x62F5987F
-Object CREATE_WEAPON_OBJECT(Hash weaponHash, int ammoCount, float x, float y, float z, BOOL showWorldModel, float scale, Any p7);
+Object CREATE_WEAPON_OBJECT(Hash weaponHash, int ammoCount, float x, float y, float z, BOOL showWorldModel, float scale, cs_type(Any) BOOL scriptHostObject);
 ```
 
-```
-Now has 8 params.  
-```
+Create a weapon object that cannot be attached to a ped. If you want to create a weapon object that can be attached to a ped, use [CREATE_OBJECT](#_0x509D5878EB39E842) instead.
 
 ```
-NativeDB Added Parameter 9: Any p8
-NativeDB Added Parameter 10: Any p9
```

Comment:
Hmm, why did you remove these annotations? 🤔

`p8` is a boolean, I will name it `bRegisterAsNetworkObject`.
`p9` is another boolean, which I will name `bScriptHostObject`.

Don't add them to the function definition/parameters since it will break ABI, add them as annotations here, thanks! 😀

Reply:
Should be good!